src/ui/pages/Menu/Menu.tsx

```diff
@@ -116,22 +134,11 @@ const Menu = () => {
     toggleEditProfile();
   };
 
+  useEffect(() => {
+    setShowConnections(selectedOption === SubMenuKey.Connections);
+  }, [selectedOption]);
+
   const submenuMapData: [SubMenuKey, SubMenuData][] = [
```

Comment:
General comment and don't fix in this PR since it's not high priority but it's a little funky to me that all the sub menus are in one list. When some are accessed from `Menu` and some from within `Settings` - just seems like they should be separate. But not important

Reply:
The problem with that is Settings is a sub-menu which is rendering more sub-elements inside, it's a nested menu, but because the entire logic for making that work is in the main Menu file there was no other way to make that work. I personally don't like that either, but maybe we can spend some more time on it and see if we can find some alternatives. i'll open a separate ticket for a spike/improvement.

Reply:
I mean it's not a problem if it's in this file - I'm just curious if they need to be in the same variable

Reply:
You're right, I noticed that too and in this PR I organized them in a way that made more sense to me, grouping all the ones for the Settings together, but in fairness that could be refactored in a much better way.

Reply:
Opened ticket for spike/improvement here: https://cardanofoundation.atlassian.net/issues/DTIS-1278